orchard-cms-video-portal/azuredeploy.json

```diff
@@ -0,0 +1,269 @@
+                "workerSize": "[parameters('appServiceWorkerSize')]",
+                "hostingEnvironment": "",
+                "numberOfWorkers": 1
+            }
+        },
+        {
+            "type": "Microsoft.Media/mediaServices",
+            "name": "[variables('mediaServicesAccountName')]",
+            "apiVersion": "2015-10-01",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "storageAccounts": [
+                    {
+                        "id": "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+                        "isPrimary": true
+                    }
+                ]
+            },
+            "resources": [],
+            "dependsOn": [
+                "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]"
+            ]
+        },
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[variables('storageAccountName')]",
+            "apiVersion": "2016-01-01",
+            "location": "[resourceGroup().location]",
+            "sku": {
+                "name": "[variables('storageAccountType')]"
+             },
+            "kind": "Storage",
+            "properties": {
+            }
+        },
+        {
+            "apiVersion": "2015-02-01",
+            "name": "[parameters('appName')]",
+            "type": "Microsoft.Web/Sites",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "[concat('hidden-related:', '/subscriptions/', subscription().subscriptionId,'/resourcegroups/', resourceGroup().name, '/providers/Microsoft.Web/serverfarms/', variables('appServicePlanName'))]": "empty"
+            },
+            "dependsOn": [
+                "[concat('Microsoft.Web/serverFarms/', variables('appServicePlanName'))]"
+            ],
+            "properties": {
+                "name": "[parameters('appName')]",
+                "serverFarmId": "[concat('/subscriptions/', subscription().subscriptionId,'/resourcegroups/',resourceGroup().id, '/providers/Microsoft.Web/serverfarms/', variables('appServicePlanName'))]",
+                "hostingEnvironment": ""
+            },
+            "resources": [
+                {
+                    "apiVersion": "2015-04-01",
+                    "name": "MSDeploy",
+                    "type": "Extensions",
+                    "dependsOn": [
+                        "[concat('Microsoft.Web/Sites/', parameters('appName'))]",
+                        "[concat('Microsoft.Sql/servers/', parameters('dbServerName'), '/databases/', parameters('dbName'))]",
+                        "[concat('Microsoft.Media/mediaservices/', variables('mediaServicesAccountName'))]"
+                    ],
+                    "properties": {
+                        "packageUri": "https://orchardamstest.blob.core.windows.net/packages/Orchard.Web.1.10.x-AMS.zip",
```

Comment:
The Web Deploy image URL is now updated to the final one: [https://auxmktplceprod.blob.core.windows.net/packages/Orchard.Web.1.10.x-AMS.zip](https://auxmktplceprod.blob.core.windows.net/packages/Orchard.Web.1.10.x-AMS.zip)